src/x/mmap/mmap.go

```diff
@@ -100,14 +100,19 @@ type Reporter interface {
 
 // FilesResult contains the result of calling MmapFiles
 type FilesResult struct {
-	Warning error
+	Descriptors map[string]Descriptor
+	Warning     error
 }
 
 // Files is a utility function for mmap'ing a group of files at once
 func Files(opener FileOpener, files map[string]FileDesc) (FilesResult, error) {
 	multiWarn := xerrors.NewMultiError()
 	multiErr := xerrors.NewMultiError()
 
+	result := FilesResult{
+		Descriptors: make(map[string]Descriptor),
```

Comment:
Hm instead of this, maybe we should just change `Bytes *[]byte` to be `Descriptor *mmap.Descriptor` in `FileDesc`. 

Not sure why I didn't do this at first.